index.js

```diff
@@ -123,6 +191,12 @@ module.exports = {
     return (this.parent && this.parent.options) || (this.app && this.app.options) || {};
   },
 
+  _getHostOptions() {
+    let host = this._findHost();
```

Comment:
The `EngineAddon` wrapper in ember-engines [overrides `_findHost`](https://github.com/ember-engines/ember-engines/blob/master/lib/engine-addon.js#L439) to point to the first lazy-loading engine it comes to rather than the root app. We may need to explicitly traverse the `parent` chain ourselves here.

Reply:
Actually, since this only ever needs the host options when `this.parent === this.project`, I think we can just access `this.app` directly and not need to bother with `findHost`.